src/app.py

```diff
@@ -2511,16 +2511,21 @@ def update_datasets_datastatus():
             if isinstance(dataset[prop], str) and \
                     len(dataset[prop]) >= 2 and \
                     dataset[prop][0] == "[" and dataset[prop][-1] == "]":
-                dataset[prop] = dataset[prop].replace("'", '"')
-                dataset[prop] = json.loads(dataset[prop])
-                if len(dataset[prop]) > 0:
-                    dataset[prop] = dataset[prop][0]
+                prop_as_list = string_helper.convert_str_literal(dataset[prop])
+                if len(prop_as_list) > 0:
+                    dataset[prop] = prop_as_list
                 else:
                     dataset[prop] = " "
```

Comment:
Related, when the converted data is a list and it's empty, we should return an empty `[]` rather than a single space. Is this " " a typo? Should be an empty string rather than a single space when the data type is string.

Reply:
Fixed. I was mistaken earlier. The issue with blank strings as opposed empty strings with some white space was caused by an earlier issue with the table that was resolved some time ago. The strings with one space in them was a relic of that. They have been fixed to be proper empty strings.